.github/workflows/SimpleBuild.yml

```diff
@@ -33,32 +33,6 @@ jobs:
 
     - name: Add Source
       run: dotnet nuget add source --username HydrologicEngineeringCenter --password ${{ secrets.GITHUB_TOKEN }} --store-password-in-clear-text --name github "https://nuget.pkg.github.com/HydrologicEngineeringCenter/index.json"
-      
-    # Restore the application to populate the obj folder with RuntimeIdentifiers
```

Comment:
Out of curiosity, why add this code in then delete?

Reply:
I only wanted to add the Add Source task to the script. It was a copy paste error that all the other stuff came with it.